src/quart/helpers.py

```diff
@@ -389,7 +389,7 @@ def _split_blueprint_path(name: str) -> List[str]:
     return bps
 
 
-def abort(code: int, *args: Any, **kwargs: Any) -> NoReturn:  # type: ignore[misc]
+def abort(code: int | Response, *args: Any, **kwargs: Any) -> NoReturn:
```

Comment:
I don't think Werkzeug's aborter can accept a Quart Response. I think this part of the API is unhelpful - what use case do you have in mind?